sdk/core/Azure.Core/src/Pipeline/ResponseClassifier.cs

```diff
@@ -32,5 +33,39 @@ public virtual bool IsErrorResponse(Response response)
             var statusKind = response.Status / 100;
             return statusKind == 4 || statusKind == 5;
         }
+
+        public static bool IsTextResponse(Response response, out Encoding encoding)
+        {
+            const string charsetMarker = "; charset=";
```

Comment:
I may be missing a detail somewhere, but wouldn't this consider `application/json` to be a non-text response type?   I believe that we would want to consider that a text response, for all intents and purposes.  

_(Source:  [RFC4627](https://www.ietf.org/rfc/rfc4627.txt), which I believe is still current and hasn't been superseded)_

Reply:
Great point!